src/fileActions.jsx

```diff
@@ -169,7 +169,7 @@ export const ForceDeleteModal = ({ selected, path, initialError }) => {
             modalTitle = cockpit.format(_("Force delete file $0?"), selectedItem.name);
         } else if (selectedItem.type === "lnk") {
             modalTitle = cockpit.format(_("Force delete link $0?"), selectedItem.name);
-        } else if (selectedItem.type === "dir" || selectedItem.items_cnt) {
+        } else if (selectedItem.type === "dir") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-256-20240212-150910-02953828-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)